hathor/conf/settings.py

```diff
@@ -373,3 +373,6 @@ def MAXIMUM_NUMBER_OF_HALVINGS(self) -> int:
     EVENT_API_DEFAULT_BATCH_SIZE: int = 100
 
     EVENT_API_MAX_BATCH_SIZE: int = 1000
+
+    # Maximum number of sync running simultaneously.
+    MAX_ENABLED_SYNC: int = 6
```

Comment:
Is there any reasoning behind the number 6?

Reply:
I changed it to 8 earlier today. It is inspired by Bitcoin's default config.